Code/Source/VehicleDynamics/DriveModels/AckermannDriveModel.cpp

```diff
@@ -69,37 +73,48 @@ namespace VehicleDynamics
+        auto currentSteeringAngle = currentSteeringElementRotation.Dot(wheelData.m_turnAxis);
+        double pidCommand = m_steeringPid.ComputeCommand(steering - currentSteeringAngle, deltaTimeNs);
+        if (AZ::IsClose(pidCommand, 0.0)) // TODO - use the third argument with some reasonable value which means "close enough"
+        {
+            return;
+        }
+
+        auto torque = pidCommand * deltaTimeSec;
+        AZ::Transform steeringElementTransform;
+        AZ::TransformBus::EventResult(steeringElementTransform, steeringEntity, &AZ::TransformBus::Events::GetWorldTM);
+        auto transformedTorqueVector = steeringElementTransform.TransformVector(wheelData.m_turnAxis * torque);
+        Physics::RigidBodyRequestBus::Event(steeringEntity, &Physics::RigidBodyRequests::ApplyAngularImpulse, transformedTorqueVector);
+    }
+
     // TODO - speed and steering handling is quite similar, possible to refactor?
-    void SimplifiedDriveModel::ApplySteering(float steering, uint64_t deltaTimeNs)
+    void AckermannDriveModel::ApplySteering(float steering, uint64_t deltaTimeNs)
     {
         if (m_steeringData.empty())
         {
             AZ_Warning("ApplySteering", false, "Cannot apply steering since no steering elements are defined in the model");
             return;
         }
 
-        const double deltaTimeSec = double(deltaTimeNs) / 1e9;
-        for (const auto& steeringElementData : m_steeringData)
-        {
-            auto steeringEntity = steeringElementData.m_steeringEntity;
-            AZ::Vector3 currentSteeringElementRotation;
-            AZ::TransformBus::EventResult(currentSteeringElementRotation, steeringEntity, &AZ::TransformBus::Events::GetLocalRotation);
-            auto currentSteeringAngle = currentSteeringElementRotation.Dot(steeringElementData.m_turnAxis);
-            double pidCommand = m_steeringPid.ComputeCommand(steering - currentSteeringAngle, deltaTimeNs);
-            if (AZ::IsClose(pidCommand, 0.0)) // TODO - use the third argument with some reasonable value which means "close enough"
-            {
-                continue;
-            }
+        auto innerSteering = atan((m_wheelbase * tan(steering)) / (m_wheelbase - 0.5 * m_track * tan(steering)));
+        auto outerSteering = atan((m_wheelbase * tan(steering)) / (m_wheelbase + 0.5 * m_track * tan(steering)));
 
-            auto torque = pidCommand * deltaTimeSec;
-            AZ::Transform steeringElementTransform;
-            AZ::TransformBus::EventResult(steeringElementTransform, steeringEntity, &AZ::TransformBus::Events::GetWorldTM);
-            auto transformedTorqueVector = steeringElementTransform.TransformVector(steeringElementData.m_turnAxis * torque);
-            Physics::RigidBodyRequestBus::Event(steeringEntity, &Physics::RigidBodyRequests::ApplyAngularImpulse, transformedTorqueVector);
+        // TODO - use proper "epsilon"
+        if (AZ::Abs(steering) > 0.01)
```

Comment:
Suggestion: move the 0.01 to a member variable. Make it accessible to the user if it makes sense to set it to different values ("steering dead zone"?)